tests/test_cli.sh

```diff
@@ -13,6 +16,22 @@ function setup_tests {
     # explicitly enable sshd for live-iso b/c it is disabled by default
     # due to security concerns (no root password required)
     sed -i.orig 's/^services.*/services --disabled="network" --enabled="NetworkManager,sshd"/' $share_dir/composer/live-iso.ks
+
+    # Make the live-iso boot more quickly (isolinux.cfg)
+    for cfg in "$share_dir"/templates.d/99-generic/live/config_files/*/isolinux.cfg; do
```

Comment:
@bcl where are these files located on RHEL 7 ? I'm not able to find them.

Reply:
`./share/live/config_files/x86/isolinux.cfg` -- rhel7 doesn't support adding new templates so it has a slightly shorter directory tree.